dashboard/src/auth/Login.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Button, Container, Form, Icon, Message, Modal } from 'semantic-ui-react';
+
+const LoginView = ({username, password, usernameChanged, passwordChanged, doLogin, error}) => (
+  <Container>
+    <Form onSubmit={doLogin}>
+      <Form.Field>
+        <label>Name</label>
+        <input focus="true" value={username} onChange={(e) => usernameChanged(e.target.value)}/>
```

Comment:
Conventionally this would be called "onUsernameChange" I think. This also helps indicate it's an event handler rather than e.g. a flag.

Reply:
fixed